Runtime/DataSchemas/RomuloExternalSchema.cs

```diff
@@ -1,39 +1,54 @@
 using Newtonsoft.Json.Linq;
+using Newtonsoft.Json.Schema;
 using ReupVirtualTwin.helpers;
```

Comment:
let's try to clean `using` statements we are not using anymore when we see them
```suggestion
using Newtonsoft.Json.Schema;
```